README.md

```diff
@@ -46,6 +46,8 @@ Usage
 8. Wait for Jenkins to restart.
 
 ## Generating a proper IAM user:
+Note : The following steps need to be followed if you are using Secret and Access key to access Device Farm.
+In case you want to access Device Farm using IAM Role, refer to "Generating a proper IAM role" section
```

Comment:
If you want to access Device Farm using IAM Role ....

Reply:
changed